wait_for_cups.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+#
+# This script waits TIMEOUT seconds for connection to cups
+# to be established and exit with 0 if success or 1 if error
+set -o pipefail
+set -o nounset
+
+TIMEOUT=$1
+
+TIMEOUT_END=$(($(date +%s) + TIMEOUT))
+result=0
+while [ $result -ne 1 ]; do
+  echo "Waiting for connection to cups..."
+  result="`lpstat -r | grep 'scheduler is running' | wc -l`"
+  if [ $result -eq 1 ]; then
```

Comment:
From our research there is not a better way to determine whether cups (lpstat printing system) is up and running than this. We can look at processes but it doesn't confirm that cups is readily available as it could be in a non-ready state like when it is starting up.
The reason we moved it from host and port is because we believe the printing web app is becoming accessible before the printing service is available meaning the rake task is running prematurely and not populating the missing printers.
In this case 'lpstat -r' returns 'scheduler is running' or 'scheduler is not running' depending on CUPS' status.